.travis.yml

```diff
@@ -10,10 +10,14 @@ branches:
 git:
   depth: 1
 
+jobs:
+  include:
+    - env: CMD="make ci e2e status=keep lighthouse=true"
+    - env: CMD="make ci e2e status=keep lighthouse=true globalnet=true"
```

Comment:
It looks like only build is validated but there are no tests run - https://travis-ci.com/github/submariner-io/submariner-operator/jobs/298846843#L1942

Please check.